.github/workflows/regression.yml

```diff
@@ -12,14 +12,11 @@ jobs:
           - 14
           - 13
           - 12
-          - 11
-          - 10
```

Comment:
Is there a reason to remove the tests? Yes, PostgreSQL version 10 and 11 is no longer supported, but it will be nice to know if and when `pg_repack` fails on these versions.